src/textarea/index.tsx

```diff
@@ -22,7 +22,7 @@ const StyledTextarea = styled(Textarea)`
     display: flex;
     box-sizing: border-box;
     color: ${color.primaryText};
-    background-color: ${color.inputBackground};
+    background-color: yellow;
```

Comment:
changed for testing purposes?

Reply:
You don't like yellow :) ?
Updated, thanks for catching :)

Reply:
👏